src/main/java/dev/snowdrop/release/services/AutoUpdateService.java

```diff
@@ -0,0 +1,165 @@
+package dev.snowdrop.release.services;
+
+import com.atlassian.jira.rest.client.api.domain.Issue;
+import dev.snowdrop.release.model.Component;
+import dev.snowdrop.release.model.Release;
+import dev.snowdrop.release.model.buildconfig.BuildConfig;
+import org.apache.commons.lang3.text.WordUtils;
+import org.jboss.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+@ApplicationScoped
+public class AutoUpdateService {
```

Comment:
The name could be better since it doesn't really explain what it is doing. What is it auto-updating?

Reply:
Renamed it to `BuildConfigUpdateService`.